Apps/Sandcastle/gallery/Particle System.html

```diff
@@ -244,7 +244,7 @@
 
     //Load the Cesium plane model to represent the entity
     model : {
-        uri : '../../SampleData/models/CesiumAir/Cesium_Air.gltf',
+        uri : '../../../../Apps/SampleData/models/CesiumAir/Cesium_Air.gltf',
```

Comment:
I don't think these need to be changed, this path should work with both the built and non-built versions of sandcastle.